.travis.yml

```diff
@@ -12,15 +12,31 @@ jobs:
   include:
     - stage: puppetserver tests
       name: "Java 8"
-      jdk: openjdk8
-    - jdk: openjdk11
-      name: "Java 11"
-    - jdk: openjdk11
+      before_install:
```

Comment:
wondering if all these should go into a script that can take the Java version as a param, so we don't have to repeat it four times.

Reply:
Yeah probably. It does look like they're identical except for the number.

Reply:
Is it possible/desirable to just run `sudo apt-get install -y openjdk-8-jdk` without first deleting the old java?

Reply:
I can try it. I copied this from what Nick did in [clj-http-client](https://github.com/puppetlabs/clj-http-client/commit/6a6f27f7be96fbcc8479fd352952476b496925c8). If it works we can do that there too I suppose.

Reply:
Apparently not 🤷‍♀️